use-cases/client-server-msgs-perf/svmsg_file_server.c

```diff
@@ -293,40 +306,28 @@ main(int argc, char *argv[])
             resp.counter = ++clientp->client_ctr;   // Do Increment
             clientp->num_ops++;                     // Track # of operations
 
-            if (clock_gettime(clock_id, &ts1)) {    // Timing ends
-                errExit("clock_gettime-ts1");
-            }
-
-            nsec0 = timespec_to_ns(&ts0);
-            nsec1 = timespec_to_ns(&ts1);
-            elapsed_ns = (nsec1 - nsec0);           // Elapsed-ns for this op
-
 #if L3_ENABLED
 
-            // Record time-consumed. (NOTE: See clarification below.)
+            // Record new-counter value, to show that something can be logged.
 #  if L3_FASTLOG_ENABLED
             l3_log_fast("Server msg: Increment: ClientID=%d, "
-                        "Thread-CPU-time=%" PRIu64 " ns. (L3-fast-log)",
-                        resp.clientId, elapsed_ns);
+                        "Elapsed time=%" PRIu64 " ns. (L3-fast-log)",
+                        resp.clientId, resp.counter);
 #  else
             l3_log("Server msg: Increment: ClientID=%d, "
-                   "Thread-CPU-time=%" PRIu64 " ns.",
-                   resp.clientId, elapsed_ns);
+                   "Elapsed time=%" PRIu64 " ns.",
+                   resp.clientId, resp.counter);
 #  endif
 
 #endif // L3_ENABLED
 
-            // Reuse variable to reacquire current TS for accumulation.
-            // Cumulative time-consumed gives a read into impact of any
-            // logging call that might be done in diff micro-benchmarks.
-
-            if (clock_gettime(clock_id, &ts1)) {    // Cumulative timing ends
-                errExit("clock_gettime-ts1");
-            }
-            nsec1 = timespec_to_ns(&ts1);
-            elapsed_ns = (nsec1 - nsec0);
-            clientp->cumu_time_ns += elapsed_ns;    // Cumulative elapsed-ns
+            break;
 
+          case REQ_MT_SET_THROUGHPUT:
```

Comment:
New op-code sent by client to register its client-side throughput metric with the server.

Server can then aggregate avg. client-side throughput across all clients, in its summary report.